### DISCUSSION TITLE
Differential Expression Analysis v3 vs. v4

### DISCUSSION THREAD

**Discussion by jenriquezl337** [satijalab/seurat#4126] [Q&A]
I noticed that the FindAllMarkers() output for v4 now has "avg_log2FC" but previously in v3 it was just "avg_logFC." From past discussion posts v3 and older versions of Seurat showed differential expression as natural log. With this log2 change in v4, does this also mean that the FeaturePlot() scale in v4 is now log2 FC or is the scale still natural log? Is there a variable within the FeaturePlot() or FindAllMarkers() function to specify the scale as log2 or natural log in v4?

**Comment by JonThom** [comment]
Hi,
check out the [Changes in Seurat v4](https://satijalab.org/seurat/articles/v4_changes.html):

> We have restructured the code of the FindMarkers() function to be easier to understand, interpret, and debug. The results of differential expression are unchanged. However, by default we now prefilter genes and report fold change using base 2, as is commonly done in other differential expression packages, instead of natural log. If the default option is set, the output of FindMarkers() will include the column avg_log2FC, instead of avg_logFC. Users can restore the previous behavior (natural log), by specifying base = exp(1).

**Reply by jenriquezl337** [reply]
Thanks for the clarification. What about the scale for FeaturePlot()? Is that still visualized as natural log values?

**Reply by timoast** [reply]
@jenriquezl337 FeaturePlot shows the normalised data, which is different to the fold-change values reported by FindMarkers. See the documentation for `NormalizeData()` to find out exactly how these values are calculated, but in short: yes, they use natural log.
